tests/test_docbuilder.py

```diff
@@ -69,8 +69,8 @@ def testMarshalDocumentPermanentId(self, docBuilder):
         assert docBuilder.marshal().get("permanentId") == "the_id"
 
     def testMarshalDocumentPermanentIdFromURI(self, docBuilder):
-        assert docBuilder.marshal().get("permanentId") == hashlib.sha256(
-            "https://foo.com".encode("utf-8")).hexdigest()
+        # 'https://foo.com'
```

Comment:
Not sure to understand what the comment is doing here 😕 ?

Reply:
It's a reminder of what the hash for the `permanentid` refers to.